src/main/java/com/google/devtools/build/lib/worker/WorkerOptions.java

```diff
@@ -46,16 +46,6 @@
       })
   public Void experimentalPersistentJavac;
 
-  @Option(
```

Comment:
You need to move the flag to the graveyard, or any Bazel invocation passing that flag will fail.

Reply:
my thought here was that since this was an undocumented flag for an alpha feature removing it would be ok, especially if that's in the 5.x release not the 4.x release. Also flipping the default wouldn't change much AFAICT since workers can only support 1 protocol at once, so flipping it back to false wouldn't be super useful for folks, still happy to make that change if you think we should, but that's why I just deleted it